x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_groups.ts

```diff
@@ -27,20 +28,22 @@ interface UseFetchSloGroupsResponse {
 export function useFetchSloGroups({
   page = 1,
   perPage = DEFAULT_SLO_GROUPS_PAGE_SIZE,
+  groupBy = 'ungrouped',
 }: SLOGroupsParams = {}): UseFetchSloGroupsResponse {
   const {
     http,
     notifications: { toasts },
   } = useKibana().services;
   const { data, isLoading, isSuccess, isError, isRefetching } = useQuery({
-    queryKey: sloKeys.groups({ page, perPage }),
+    queryKey: sloKeys.groups({ page, perPage, groupBy }),
```

Comment:
I feel like we're gonna start confusing this `groupBy` with `slo.groupBy`, especially new joiners. Not sure about the solution yet but something to think about.

And I have a PR too that introduces another concept called 'groupings', so just a lot of things that have the word group in the name. 😅